centaur/src/main/resources/standardTestCases/virtual_private_cloud/check_network_in_vpc.wdl

```diff
@@ -2,12 +2,17 @@ version 1.0
 
 task get_network {
   command {
+    set -euo pipefail
+
     apt-get install --assume-yes jq > /dev/null
     INSTANCE=$(curl -s "http://metadata.google.internal/computeMetadata/v1/instance/name" -H "Metadata-Flavor: Google")
     ZONE=$(curl -s "http://metadata.google.internal/computeMetadata/v1/instance/zone" -H "Metadata-Flavor: Google" | sed -E 's!.*/(.*)!\1!')
     TOKEN=$(gcloud auth application-default print-access-token)
     INSTANCE_METADATA=$(curl "https://www.googleapis.com/compute/v1/projects/broad-dsde-cromwell-dev/zones/$ZONE/instances/$INSTANCE" -H "Authorization: Bearer $TOKEN" -H 'Accept: application/json')
-    echo $INSTANCE_METADATA | jq -r '.networkInterfaces[0].network' | sed -E 's!.*/(.*)!\1!'
+    NETWORK_OBJECT=`echo $INSTANCE_METADATA | jq -r '.networkInterfaces[0]'`
```

Comment:
```shell
NETWORK_OBJECT=$(echo $INSTANCE_METADATA | jq  --exit-status --raw-output '.networkInterfaces[0]')
```

Suggestions:
- Use `$()` instead of backticks, as the former may be [nested](https://stackoverflow.com/questions/9449778/what-is-the-benefit-of-using-instead-of-backticks-in-shell-scripts#answer-9449788)
- Get in the habit of adding `--exit-status` to `jq` to stop it from [silently exiting without error](https://stedolan.github.io/jq/manual/v1.6/#Invokingjq)
- (Debatable) Short args like `-r` are meant for [typing quickly on the command line](https://superuser.com/questions/174564/why-are-there-short-and-long-alternatives-for-command-line-options)

Same goes for the other `jq` calls on the lines below.

Bonus ToL: If `echo ... | jq ...` is working, leave it. But in general I don't know if I [trust `echo` not to modify the contents of variables](https://stackoverflow.com/questions/40027395/passing-bash-variable-to-jq-select#comment67406349_40027637). I don't have the exact change ready at the moment, but there are discussions around the web of [other ways of inputing values into `jq`](https://stackoverflow.com/questions/47105490/can-i-pass-a-string-variable-to-jq-not-the-file#answer-47106137).

Reply:
Modified as suggested. Thank you!